.github/workflows/misspellings_checks.yml

```diff
@@ -0,0 +1,28 @@
+name: Misspellings checks
+
+# This workflow performs some misspelling checks on the repository
+# It is there to help us maintain a level of quality in our codebase and does not have to be kept on forked
+# applications.
```

Comment:
Maybe update the comment ? (weird to talk about forked apps here ^^)

Reply:
Done :wink: